packages/xarray/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: xarray
+  version: 2022.3.0
+source:
+  url: https://files.pythonhosted.org/packages/4e/6e/2d2430e021fcce47771dcb168599b3d71f26f63a89bd3ff78f56759f1701/xarray-2022.3.0-py3-none-any.whl
+  sha256: 560f36eaabe7a989d5583d37ec753dd737357aa6a6453e55c80bb4f92291a69e
+requirements:
+  run:
+    # - importlib_metadata
+    - numpy
+    - packaging
+    - pandas
+    # - typing_extensions
```

Comment:
> You need to require `typing-extensions` with a dash instead of an underscore.

Thanks for the pointer @hoodmane. I'm actually not 100% sure if `typing-extensions` is needed at runtime. According to https://github.com/pydata/xarray/blob/9b4d0b29c319f4b68f89328b1bf558711f339504/setup.cfg#L77-L80, it seems like only `numpy`, `pandas` and `packaging` are required? It might just be that `xarray`'s conda-forge `recipe/meta.yaml` file hasn't been updated for a while.

Reply:
So it seems to work without `typing-extensions`? `test_packages_common.py::import[firefox-xarray] PASSED                   [ 43%]`, see https://app.circleci.com/pipelines/github/pyodide/pyodide/4083/workflows/39debe58-bf65-4fe9-a1dc-4b96350cf193/jobs/45126?invite=true#step-102-157

Reply:
Yes, it's an optional dependency for py <3.10 https://github.com/pydata/xarray/blob/1941695ade6a3e4e46a07e6604f29bda5176de26/xarray/core/utils.py#L300 so it should be fine not to include it.